tokens-studio/figma-plugin
# fix applying typography tokens variable values

### Why does this PR exist?

Closes #3215 



### What does this pull request do?

Addresses the concern where a typography token is applied with exported variables and the applied variable is not displayed in Figma.



### Testing this change

- Create a typography token
- Export as variable
- Apply the token on a text node
- The corresponding variable of the applied token should be displayed in Figma



### Additional Notes (if any)
